### ISSUE TITLE
Fix Mac App Store compatibility

### ISSUE THREAD

**Issue by msavin** [wojtkowiak/meteor-desktop#222]
Hey man! 

I haven't been able to reach ya through email, hope everything is okay. I was wondering if you had any news on Mac App Store compatibility? I really need this to get my app up.

If you can push up the 2.0.1 branch, I would be happy to try it. Maybe you can publish it as 0.0.2 just so that it would not compete with the stable 2.0.0?

**Comment by Ajaay** [comment]
I think this one could come to bite us soon as well, so happy to help here where I can with testing etc

**Comment by msavin** [comment]
Yeah I am months behind on publishing a newly completed app. I think
somebody is trying to get it on the Meteor forum, they commented on the
meteor-desktop thread.

There’s also a 2.0.1 branch which may get it working, but I don’t know how
to get it running.

**Comment by msavin** [comment]
@wojtkowiak beep - willing to pay for this to expedited 😇

**Comment by msavin** [comment]
I haven't tried this yet, but one company had success with it:
https://forums.meteor.com/t/announcing-meteor-desktop/31111/78
